### PR TITLE
Switching to builtins

### PR DESCRIPTION
**Pull request by saadrahim**

### CODE REVIEW

File: rocprim/include/rocprim/intrinsics/warp.hpp [hunk 1]
```diff
@@ -47,8 +47,13 @@ ROCPRIM_DEVICE inline
 unsigned int masked_bit_count(unsigned long long x, unsigned int add = 0)
 {
     int c;
+    #ifdef __HIP__
+    c = ::__builtin_amdgcn_mbcnt_lo(static_cast<int>(x), add);
+    c = ::__builtin_amdgcn_mbcnt_hi(static_cast<int>(x >> 32), c);
+    #else
```
Comment (saadrahim) [inline rocprim/include/rocprim/intrinsics/warp.hpp:53]: is this formatted right?

Comment (eidenyoshida) [reply rocprim/include/rocprim/intrinsics/warp.hpp:53]: I think so. Have you confirmed that it flows down that path?

Comment (saadrahim) [reply rocprim/include/rocprim/intrinsics/warp.hpp:53]: We don't have a test for this. I was not told the failing build yet

Comment (saadrahim) [reply rocprim/include/rocprim/intrinsics/warp.hpp:53]: Flying blind unfortunately

**Comment by saadrahim** [comment]
@neon60 Can you let me know if this code path is included in our tests? We really need a tool to get test coverage.

**Comment by saadrahim** [comment]
Can someone approve this or suggest how to test it?

**Pull request merged**